migration/i7.1z/oracle/202009111500_IDEMPIERE-3981.sql

```diff
@@ -0,0 +1,1036 @@
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Order Column', Description='Column determining the order', Help='Integer Column of the table determining the order (display, sort, ..). If defined, the Order By replaces the default Order By clause. It should be fully qualified (i.e. "tablename.columnname").', SeqNo=200, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL, Placeholder=NULL,Updated=TO_DATE('2019-06-03 10:16:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=5709
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Included Column', Description='Column determining if a Table Column is included in Ordering', Help='If a Included Column is defined, it decides, if a column is active in the ordering - otherwise it is determined that the Order Column has a value of one or greater', SeqNo=210, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL, Placeholder=NULL,Updated=TO_DATE('2019-06-03 10:16:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=5708
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Link Column', Description='Link Column for Multi-Parent tables', Help='The Link Column indicates which column is the primary key for those situations where there is more than one parent.  Only define it, if the table has more than one parent column (e.g. AD_User_Roles).', SeqNo=220, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL, Placeholder=NULL,Updated=TO_DATE('2019-06-03 10:16:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=1546
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Parent Column', Description='The link column on the parent tab.', Help=NULL, SeqNo=230, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL, Placeholder=NULL,Updated=TO_DATE('2019-06-03 10:16:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=57266
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Process', Description='Process or Report', Help='The Process field identifies a unique Process or Report in the system.', SeqNo=240, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL, Placeholder=NULL,Updated=TO_DATE('2019-06-03 10:16:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=2575
+;
+
+-- Jun 3,  2019 10:16:32 BRT
+UPDATE AD_Field SET Name='Display Logic', Description='If the Field is displayed, the result determines if the field is actually displayed', Help='format := {expression} [{logic} {expression}]<br>
+expression := @{context}@{operand}{value} or @{context}@{operand}{value}<br>
+logic := {|}|{&}<br>
+context := any global or window context <br>
+value := strings or numbers<br>
+logic operators	:= AND or OR with the previous result from left to right <br>
+operand := eq{=}, gt{&gt;}, le{&lt;}, not{~^!} <br>
+Examples: <br>
+@AD_Table_ID@=14 | @Language@!GERGER <br>
```

Comment:
lines starting with @ throw error in oracle, like:
  9  SP2-0310: unable to open file "PriceLimitxe>10.sql"
  9  SP2-0310: unable to open file "Namexe>J<br>.sql"

This error happens on the lines starting with:
@AD_Table_ID@=14 | @Language@!GERGER <br>
@PriceLimit@>10 | @PriceList@>@PriceActual@<br>
@Name@>J<br>

They are solved changing those lines to:
'||chr(64)||'AD_Table_ID@=14 | @Language@!GERGER <br>
'||chr(64)||'PriceLimit@>10 | @PriceList@>@PriceActual@<br>
'||chr(64)||'Name@>J<br>
respectively

See for example: migration/i6.2z/oracle/201906261250_IDEMPIERE-3996.sql

There are 9 op these lines in this script